src/plugins/vis_type_timelion/public/components/timelion_vis.scss

```diff
@@ -1,60 +1,13 @@
-.timChart {
+.timelionChart {
   height: 100%;
   width: 100%;
   display: flex;
   flex-direction: column;
-
-  // Custom Jquery FLOT / schema selectors
-  // Cannot change at the moment
-
-  .chart-top-title {
-    @include euiFontSizeXS;
-    flex: 0;
-    text-align: center;
-    font-weight: $euiFontWeightBold;
-  }
-
-  .chart-canvas {
-    min-width: 100%;
-    flex: 1;
-    overflow: hidden;
-  }
-
-  .legendLabel {
-    white-space: nowrap;
-    text-overflow: ellipsis;
-    overflow-x: hidden;
-    line-height: normal;
-  }
-
-  .legendColorBox {
-    vertical-align: middle;
-  }
-
-  .ngLegendValue {
-    color: $euiTextColor;
-    cursor: pointer;
-
-    &:focus,
-    &:hover {
-      text-decoration: underline;
-    }
-  }
-
-  .ngLegendValueNumber {
-    margin-left: $euiSizeXS;
-    margin-right: $euiSizeXS;
-    font-weight: $euiFontWeightBold;
-  }
-
-  .flot-tick-label {
-    font-size: $euiFontSizeXS;
-    color: $euiColorDarkShade;
-  }
 }
 
-.timChart__legendCaption {
-  color: $euiTextColor;
-  white-space: nowrap;
+.timelionChart__topTitle {
```

Comment:
Two questions regarding the element using this class:

1. Would it make sense to change the element using this class to be an `EuiTitle` component, with a prop of `size="xxxs"` and a child heading element (`h1`–`h6`; whichever makes the most sense for the hierarchy)? Doing so would remove the need for all of these styles, except the `text-align: center;` and it would be a bit more semantic.

2. I noticed that when no title is applied, an empty `div.timelionChart__topTitle` remains on the page regardless. Would it be possible not to render this element if no title exists?

Reply:
You are right. Now I use `EuiTitle` and don't render element if no title exists.